doc/source/examples/dem/granular-mixer/granular-mixer.rst

```diff
@@ -0,0 +1,144 @@
+==================================
+Granular Mixer
+==================================
+
+This example simulates the packing and mixing of particles in a mixer with a pitched-blade impeller. It is recommended to visit `DEM parameters <../../../parameters/dem/dem.html>`_ for more detailed information on the concepts and physical meanings of the parameters in Lethe-DEM.
+
+Features
+----------------------------------
+- Solvers: ``dem_3d``
+- Floating mesh
+- Gmsh grids
+- Bidispersed particles
```

Comment:
```suggestion
- Bidispersed particle size
```

Reply:
If they have the same size, I would instead specify something like particles with different ids or something like that.

Reply:
Added.